docs/advanced-features/react-18/switchable-runtime.md

```diff
@@ -8,19 +8,6 @@ Next.js has two **server runtimes** where you can render parts of your applicati
 
 By default, Next.js uses the Node.js runtime. [Middleware](https://nextjs.org/docs/advanced-features/middleware) and [Edge API Routes](https://nextjs.org/docs/api-routes/edge-api-routes) use the Edge runtime.
 
-## Global Runtime Option
```

Comment:
isn't this still valid? I mean, although it's out of experimental, a global configuration example could be convenient here.